tailor_distro/manage/__init__.py

```diff
@@ -19,10 +20,24 @@ def main():
 
     args = vars(parser.parse_args())
 
+    # TODO(dlu): Maybe switch to add_mutually_exclusive_group
+    if args['rosdistro_path'] != '.' and args['rosdistro_url']:
+        raise RuntimeError('Cannot specify rosdistro using both path and url.')
+
+    distro_name = args.pop('distro')
+
+    if args['rosdistro_url']:
+        rosdistro_repo = RemoteROSDistro(args.pop('rosdistro_url'), args.pop('rosdistro_branch'), distro_name)
+        args.pop('rosdistro_path')
+    else:
+        rosdistro_repo = LocalROSDistro(args.pop('rosdistro_path'), distro_name)
+        args.pop('rosdistro_url')
+        args.pop('rosdistro_branch')
+
     verb = args.pop('verb')
 
     if verb is not None:
-        sys.exit(verb(**args))
+        sys.exit(verb(rosdistro_repo, **args))
```

Comment:
may want to pass this in as a kwarg, since **args is decomposed into kwargs as well

Reply:
You mean just 
```suggestion
        sys.exit(verb(rosdistro_repo=rosdistro_repo, **args))
```
?

Reply:
looks good. it's scary to mix positional and keyword args for something like this, where each verb defines the arg order independently